portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,8 +36,20 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println(convertToJsonUsingGson(comments));
+    
+    Query query = new Query("Task");
```

Comment:
Should this be called "Comment" instead?
Also you should probably set this up as a Java constant.